src/lib.rs

```diff
@@ -230,6 +231,19 @@ pub enum ConfigLevel {
     Highest,
 }
 
+/// An enumeration for favoring sides to resolve merge conflicts
+#[derive(PartialEq, Eq, Debug, Copy, Clone)]
+pub enum FileFavor {
+    /// Record conflicts in the index without favoring sides
```

Comment:
Could these docs copy over the longer form docs in the corresponding header file?